### PR TITLE
Multishop product basic

### PR DESCRIPTION
**Pull request by jolelievre**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Handle multishop modification for basic inputs
| Type?             | improvement
| Category?         | BO
| BC breaks?        | yes
| Deprecations?     | no
| Fixed ticket?     | ~
| How to test?      | ~
| Possible impacts? | The button id in dropdown translatable widget needed to be modified If some JS code targets it the selector will not work anymore

### Breaking Change

In the translatable widget, the form ID was assigned on the dropdown button, which made impossible to target the div container (which is the element that should have this ID). The ID was moved to the widget div and the button ID has been modified by appending `_dropdown` to the form ID. Which means if some js code was targeting this button (or a CSS rule) the selector won't work as expected anymore.

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/27584)

### CODE REVIEW

File: src/Adapter/Product/Repository/ProductMultiShopRepository.php [hunk 1]
```diff
@@ -169,6 +170,10 @@ public function getProductDefaultShopId(ProductId $productId): ShopId
      */
     public function getByShopConstraint(ProductId $productId, ShopConstraint $shopConstraint): Product
     {
+        if ($shopConstraint->getShopGroupId()) {
+            throw new InvalidShopConstraintException('Product has no features related with shop group use single shop and all shops constraints');
```
Comment (kpodemski) [inline src/Adapter/Product/Repository/ProductMultiShopRepository.php:174]: > use single shop and all shops constraints

Did you mean: use single shop OR all shops constraints?

**Review by kpodemski — APPROVED** [review]

File: src/Adapter/Product/Repository/ProductMultiShopRepository.php (hunk 1 above)
Comment (jolelievre) [reply src/Adapter/Product/Repository/ProductMultiShopRepository.php:174]: Indeed or would be more adapted, do you want me to fix it?

Comment (kpodemski) [reply src/Adapter/Product/Repository/ProductMultiShopRepository.php:174]: no need 👍

**Review by matthieu-rolland — APPROVED** [review]

**Review by NeOMakinG — APPROVED** [review]

**Comment by NeOMakinG** [comment]
QA by dev?

**Comment by jolelievre** [comment]
@NeOMakinG no need for QA on this one, it is part of a much larger feature split into multiple PRs So to win a bit of time we merge them as they are reviewed and a global QA will be performed on all the tabs

This has been negotiated and agreed with @Robin-Fischer-PS

**Pull request merged**